src/components/Facility/ConsultationDoctorNotes/index.tsx

```diff
@@ -54,53 +55,48 @@ const ConsultationDoctorNotes = (props: ConsultationDoctorNotesProps) => {
 
   const initialData: PatientNoteStateType = {
     notes: [],
-    cPage: 1,
-    totalPages: 1,
     facilityId: facilityId,
     patientId: patientId,
   };
   const [state, setState] = useState(initialData);
 
-  const onAddNote = async () => {
+  const { mutate: addNote } = useAddPatientNote({
+    patientId,
+    thread,
+    consultationId,
+  });
+
+  const onAddNote = () => {
     if (!/\S+/.test(noteField)) {
       Notification.Error({
         msg: "Note Should Contain At Least 1 Character",
       });
       return;
     }
-
-    const { res } = await request(routes.addPatientNote, {
-      pathParams: {
-        patientId: patientId,
-      },
-      body: {
-        note: noteField,
-        thread,
-        consultation: consultationId,
-        reply_to: reply_to?.id,
-      },
+    setReplyTo(undefined);
+    setNoteField("");
+    addNote({
+      note: noteField,
+      reply_to: reply_to?.id,
+      thread,
+      consultation: consultationId,
     });
-
-    if (res?.status === 201) {
-      Notification.Success({ msg: "Note added successfully" });
-      setState({ ...state, cPage: 1 });
-      setNoteField("");
-      setReload(true);
-      setReplyTo(undefined);
-    }
   };
 
-  useTanStackQueryInstead(routes.getPatient, {
-    pathParams: { id: patientId },
-    onResponse: ({ data }) => {
-      if (data) {
-        setPatientActive(data.is_active ?? true);
-        setPatientName(data.name ?? "");
-        setFacilityName(data.facility_object?.name ?? "");
-      }
-    },
+  const { data } = useQuery({
+    queryKey: ["/patient", patientId, thread],
+    queryFn: query(routes.getPatient, {
+      pathParams: { patientId },
+      queryParams: { thread, offset: thread },
```

Comment:
do a self review of your code before requesting review. why would thread and offset: thread be query params for getting the patient object? makes no sense.